src/server/firefox/ffPage.ts

```diff
@@ -258,7 +257,8 @@ export class FFPage implements PageDelegate {
   }
 
   _onScreencastStarted(event: Protocol.Page.screencastStartedPayload) {
-    this._browserContext._browser._videoStarted(event.screencastId, event.file, this.pageOrError());
+    const video = new Video(this._browserContext, event.screencastId, event.file);
```

Comment:
Let's revert this change, the video creation is now repeated 3 times for no good reason.

Reply:
Done.